dask/blockwise.py

```diff
@@ -236,17 +240,177 @@ def blockwise(
 ):
     """Create a Blockwise symbolic mutable mapping
 
-    This is like the ``make_blockwise_graph`` function, but rather than construct a
-    dict, it returns a symbolic Blockwise object.
+    Applies a task, ``func``, across blocks from many different input
+    collections.  We arrange the pattern with which those blocks interact with
+    sets of matching indices.  E.g.::
+
+        make_blockwise_graph(task, 'z', 'i', 'x', 'i', 'y', 'i')
+
+    yield an embarrassingly parallel communication pattern and is read as
```

Comment:
having those doc strings for `make_blockwise_graph` was a little off. We now only have this function as public interface so I moved the docs over here and migrated the doctests to the current version. Some of them are a little tedious so I skipped them but overall this should be working